--- cpmtools-2.13/Makefile.in.orig	2010-03-30 19:53:11.000000000 +0200
+++ cpmtools-2.13/Makefile.in	2010-11-14 21:42:47.443102452 +0100
@@ -80,24 +80,24 @@
 		-./fsck.cpm -f pcw -n badfs/label
 
 install:	all
-		$(INSTALL) -s -m 755 cpmls $(BINDIR)/cpmls
-		$(INSTALL) -s -m 755 cpmcp $(BINDIR)/cpmcp
-		$(INSTALL) -s -m 755 cpmrm $(BINDIR)/cpmrm
-		$(INSTALL) -s -m 755 cpmchmod $(BINDIR)/cpmchmod
-		$(INSTALL) -s -m 755 cpmchattr $(BINDIR)/cpmchattr
-		$(INSTALL) -s -m 755 mkfs.cpm $(BINDIR)/mkfs.cpm
-		$(INSTALL) -s -m 755 fsck.cpm $(BINDIR)/fsck.cpm
-		[ $(FSED_CPM) != '' ] && $(INSTALL) -s -m 755 fsed.cpm $(BINDIR)/fsed.cpm
-		$(INSTALL_DATA) diskdefs @datarootdir@/diskdefs
-		$(INSTALL_DATA) cpmls.1 $(MANDIR)/man1/cpmls.1
-		$(INSTALL_DATA) cpmcp.1 $(MANDIR)/man1/cpmcp.1
-		$(INSTALL_DATA) cpmrm.1 $(MANDIR)/man1/cpmrm.1
-		$(INSTALL_DATA) cpmchmod.1 $(MANDIR)/man1/cpmchmod.1
-		$(INSTALL_DATA) cpmchattr.1 $(MANDIR)/man1/cpmchattr.1
-		$(INSTALL_DATA) mkfs.cpm.1 $(MANDIR)/man1/mkfs.cpm.1
-		$(INSTALL_DATA) fsck.cpm.1 $(MANDIR)/man1/fsck.cpm.1
-		$(INSTALL_DATA) fsed.cpm.1 $(MANDIR)/man1/fsed.cpm.1
-		$(INSTALL_DATA) cpm.5 $(MANDIR)/man5/cpm.5
+		$(INSTALL) -D -m 755 cpmls $(DESTDIR)$(BINDIR)/cpmls
+		$(INSTALL) -D -m 755 cpmcp $(DESTDIR)$(BINDIR)/cpmcp
+		$(INSTALL) -D -m 755 cpmrm $(DESTDIR)$(BINDIR)/cpmrm
+		$(INSTALL) -D -m 755 cpmchmod $(DESTDIR)$(BINDIR)/cpmchmod
+		$(INSTALL) -D -m 755 cpmchattr $(DESTDIR)$(BINDIR)/cpmchattr
+		$(INSTALL) -D -m 755 mkfs.cpm $(DESTDIR)$(BINDIR)/mkfs.cpm
+		$(INSTALL) -D -m 755 fsck.cpm $(DESTDIR)$(BINDIR)/fsck.cpm
+		[ $(FSED_CPM) != '' ] && $(INSTALL) -D -m 755 fsed.cpm $(DESTDIR)$(BINDIR)/fsed.cpm
+		$(INSTALL_DATA) -D diskdefs $(DESTDIR)@datarootdir@/diskdefs
+		$(INSTALL_DATA) -D cpmls.1 $(DESTDIR)$(MANDIR)/man1/cpmls.1
+		$(INSTALL_DATA) -D cpmcp.1 $(DESTDIR)$(MANDIR)/man1/cpmcp.1
+		$(INSTALL_DATA) -D cpmrm.1 $(DESTDIR)$(MANDIR)/man1/cpmrm.1
+		$(INSTALL_DATA) -D cpmchmod.1 $(DESTDIR)$(MANDIR)/man1/cpmchmod.1
+		$(INSTALL_DATA) -D cpmchattr.1 $(DESTDIR)$(MANDIR)/man1/cpmchattr.1
+		$(INSTALL_DATA) -D mkfs.cpm.1 $(DESTDIR)$(MANDIR)/man1/mkfs.cpm.1
+		$(INSTALL_DATA) -D fsck.cpm.1 $(DESTDIR)$(MANDIR)/man1/fsck.cpm.1
+		$(INSTALL_DATA) -D fsed.cpm.1 $(DESTDIR)$(MANDIR)/man1/fsed.cpm.1
+		$(INSTALL_DATA) -D cpm.5 $(DESTDIR)$(MANDIR)/man5/cpm.5
 
 clean:
 		rm -f *$(OBJEXT)
