--- cpmtools-2.15/device_libdsk.c.orig	2012-03-18 19:18:16.000000000 +0100
+++ cpmtools-2.15/device_libdsk.c	2013-03-02 21:24:38.201841462 +0100
@@ -31,7 +31,7 @@
   this->sectrk=sectrk;
   this->tracks=tracks;
   /* Must be an even multiple of sector size */
-  assert((offset%secLength==0);
+  assert(offset%secLength==0);
   this->offset=offset;
   
   this->geom.dg_secsize   = secLength;
@@ -65,7 +65,7 @@
 const char *Device_readSector(const struct Device *this, int track, int sector, char *buf)
 {
   dsk_err_t e;
-  e = dsk_lread(this->dev, &this->geom, buf, (track * this->sectrk) + sector + offset/this->secLength);
+  e = dsk_lread(this->dev, &this->geom, buf, (track * this->sectrk) + sector + this->offset/this->secLength);
   return (e?dsk_strerror(e):(const char*)0);
 }
 /*}}}*/
@@ -73,7 +73,7 @@
 const char *Device_writeSector(const struct Device *this, int track, int sector, const char *buf)
 {
   dsk_err_t e;
-  e = dsk_lwrite(this->dev, &this->geom, buf, (track * this->sectrk) + sector + offset/this->secLength);
+  e = dsk_lwrite(this->dev, &this->geom, buf, (track * this->sectrk) + sector + this->offset/this->secLength);
   return (e?dsk_strerror(e):(const char*)0);
 }
 /*}}}*/
