--- cpmtools-2.24/Makefile.in.orig	2023-09-12 21:55:42.577982381 +0200
+++ cpmtools-2.24/Makefile.in	2023-09-12 22:04:32.638444131 +0200
@@ -84,13 +84,14 @@ install:	all
 		[ -d $(DESTDIR)$(MANDIR)/man1 ] || $(INSTALL) -m 755 -d $(DESTDIR)$(MANDIR)/man1
 		[ -d $(DESTDIR)$(MANDIR)/man5 ] || $(INSTALL) -m 755 -d $(DESTDIR)$(MANDIR)/man5
 		[ -d $(DESTDIR)$(BINDIR) ] || $(INSTALL) -m 755 -d $(DESTDIR)$(BINDIR)
-		$(INSTALL) -s -m 755 cpmls $(DESTDIR)$(BINDIR)/cpmls
-		$(INSTALL) -s -m 755 cpmcp $(DESTDIR)$(BINDIR)/cpmcp
-		$(INSTALL) -s -m 755 cpmrm $(DESTDIR)$(BINDIR)/cpmrm
-		$(INSTALL) -s -m 755 cpmchmod $(DESTDIR)$(BINDIR)/cpmchmod
-		$(INSTALL) -s -m 755 cpmchattr $(DESTDIR)$(BINDIR)/cpmchattr
-		$(INSTALL) -s -m 755 mkfs.cpm $(DESTDIR)$(BINDIR)/mkfs.cpm
-		$(INSTALL) -s -m 755 fsck.cpm $(DESTDIR)$(BINDIR)/fsck.cpm
+		[ -d $(DESTDIR)@datarootdir@ ] || $(INSTALL) -m 755 -d $(DESTDIR)@datarootdir@
+		$(INSTALL) -m 755 cpmls $(DESTDIR)$(BINDIR)/cpmls
+		$(INSTALL) -m 755 cpmcp $(DESTDIR)$(BINDIR)/cpmcp
+		$(INSTALL) -m 755 cpmrm $(DESTDIR)$(BINDIR)/cpmrm
+		$(INSTALL) -m 755 cpmchmod $(DESTDIR)$(BINDIR)/cpmchmod
+		$(INSTALL) -m 755 cpmchattr $(DESTDIR)$(BINDIR)/cpmchattr
+		$(INSTALL) -m 755 mkfs.cpm $(DESTDIR)$(BINDIR)/mkfs.cpm
+		$(INSTALL) -m 755 fsck.cpm $(DESTDIR)$(BINDIR)/fsck.cpm
 		[ "$(FSED_CPM)" = '' ] || $(INSTALL) -s -m 755 fsed.cpm $(DESTDIR)$(BINDIR)/fsed.cpm
 		$(INSTALL_DATA) diskdefs $(DESTDIR)@datarootdir@/diskdefs
 		$(INSTALL_DATA) cpmls.1 $(DESTDIR)$(MANDIR)/man1/cpmls.1
